platform/marvell-armhf/prestera/debian/rules

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/make -f
+
+include /usr/share/dpkg/pkg-info.mk
+
+PACKAGE_PRE_NAME := mrvlprestera
+KVERSION   ?= $(shell uname -r)
+KERNEL_SRC :=  /lib/modules/$(KVERSION)
+MOD_SRC_DIR:= $(shell pwd)
+MODULE_DIR := mrvl-prestera/cpssEnabler/linuxNoKernelModule/drivers
+
+%:
+	dh $@ --with systemd,python2,python3 --buildsystem=pybuild
+
+clean:
+	dh_testdir
+	dh_testroot
+	dh_clean
+
+build:
+	# get sources
+	rm -rf mrvl-prestera || true
+	git clone https://github.com/Marvell-switching/mrvl-prestera.git
```

Comment:
we need tag to checkout a specific commit.

otherwise, the build is not reproducible.

Reply:
Ok sure, I will add git tag, and update here accordingly

Reply:
Addressed your comment, by adding git tag, and updated the git clone url with the TAG.